10c-spatial-transcriptomics.Rmd

```diff
@@ -1,98 +1,158 @@
 
-1. **Spatially Resolved Capture of RNA Molecules:** In this step, the tissue sample is typically placed on a solid substrate, such as a glass slide or a microarray chip. One common approach is to use spatially barcoded arrays or grids, where each region corresponds to a specific spatial location. Alternatively, techniques like in situ sequencing or imaging-based methods can be used. These approaches allow the capture of RNA molecules while retaining spatial information.
+2.  **Spatially Resolved Capture of RNA Molecules:** In this step, the tissue sample is typically placed on a solid substrate, such as a glass slide or a microarray chip. One common approach is to use spatially barcoded arrays or grids, where each region corresponds to a specific spatial location. Alternatively, techniques like in situ sequencing or imaging-based methods can be used. These approaches allow the capture of RNA molecules while retaining spatial information.
 
-1. **RNA Extraction and Amplification:** After spatial capture, the captured RNA molecules need to be extracted and amplified to generate sufficient material for downstream analysis. This step involves lysing the cells, extracting the RNA, and performing reverse transcription to convert RNA into complementary DNA (cDNA). Amplification methods such as PCR or in vitro transcription (IVT) are employed to generate amplified cDNA libraries.
+3.  **RNA Extraction and Amplification:** After spatial capture, the captured RNA molecules need to be extracted and amplified to generate sufficient material for downstream analysis. This step involves lysing the cells, extracting the RNA, and performing reverse transcription to convert RNA into complementary DNA (cDNA). Amplification methods such as PCR or in vitro transcription (IVT) are employed to generate amplified cDNA libraries.
 
-1. **Library Preparation:** The amplified cDNA libraries are then prepared for sequencing. This involves adding specific sequencing adapters and barcodes to each library, which enable sample multiplexing and identification during sequencing.
+4.  **Library Preparation:** The amplified cDNA libraries are then prepared for sequencing. This involves adding specific sequencing adapters and barcodes to each library, which enable sample multiplexing and identification during sequencing.
 
-1. **Sequencing:** The prepared libraries are subjected to high-throughput sequencing using platforms like Illumina or Ion Torrent. The choice of sequencing platform depends on the desired read length, throughput, and cost considerations.
+5.  **Sequencing:** The prepared libraries are subjected to high-throughput sequencing using platforms like Illumina or Ion Torrent. The choice of sequencing platform depends on the desired read length, throughput, and cost considerations.
```

Comment:
Oh, just so you know, in markdown generally, if you put `1.` for a list, it will auto-populate the correct number upon rendering.

Reply:
Ok, thank you!